test/Benchmarks/src/Map/Hash_Map.enso

```diff
@@ -43,15 +46,23 @@ collect_benches = Bench.build builder->
 
     builder.group ("Enso_Hash_Map_" + n.to_text) options group_builder->
         # Scenario similar to what is done in distinct
+        # 'Sparse' ints have fewer duplicates and cause more inserts to happen.
+        group_builder.specify "Enso_Sparse_Incremental" <|
+            Scenario.Instance (_ -> Dictionary.empty) . run_distinct data.sparse_ints
+        group_builder.specify "Java_Sparse_Incremental" <|
+            Scenario.Instance (_ -> JavaHashMapWrapper.new) . run_distinct data.sparse_ints
+
+        # Scenario similar to what is done in distinct
+        # 'Dense' ints have more duplicates and cause fewer inserts to happen.
```

Comment:
I'd add
```suggestion
        # Scenario similar to what is done in distinct
        # 'Dense' ints have more duplicates and cause fewer inserts, but more key updates to happen.
```
(assuming it's true, but it feels it should be)

Reply:
Done